### PR TITLE
Added swaggo docs to provisionerapi

### PR DESCRIPTION
**Pull request by jilleJr**
- \[x] I've added a new note in the `CHANGELOG.md` file, according to docs:
  https://iver-wharf.github.io/#/development/changelogs/writing-changelogs

## Summary

- Added swaggo dependency
- Added `swag` and `swag-force` targets to `Makefile`
- Added swaggo godocs to `pkg/provisionerapi` endpoint handlers

## Motivation

I've copied most code from wharf-api

I'm letting the "already generated" part be handled by the Makefile automatically by a non-phony target instead of our custom code we've used in wharf-api. Example:

```console
$ make swag
make: Nothing to be done for 'swag'.
```

## Preview

Screenshot from 2022-02-28 09-31-55

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,8 +20,21 @@ tidy:
 deps:
 	go install github.com/mgechev/revive@latest
 	go install golang.org/x/tools/cmd/goimports@latest
+	go install github.com/swaggo/swag/cmd/swag@v1.8.0
```
Comment (Alexamakans) [inline Makefile:23]: How come `go.mod` and `CHANGELOG.md` says `v1.7.9` when this is `v1.8.0`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -67,17 +71,17 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
...
+  - `k8s.io/client-go` from v0.0.0 to v0.23.3 (#8)
+  - `sigs.k8s.io/yaml` from v1.1.0 to v1.2.0 (#8)
+  - `github.com/gin-gonic/gin` from v1.7.1 to v1.7.7 (#59)
 
-- Removed dependencies: (#8)
+- Removed dependencies:
 
-  - `github.com/gin-gonic/gin`
-  - `github.com/go-git/go-git`
+  - `github.com/go-git/go-git` (#8)
```
Comment (Alexamakans) [inline CHANGELOG.md:84]: Could be combined with the above `Removed dependencies` entry.

File: pkg/provisionerapi/worker.go [hunk 3]
```diff
@@ -38,6 +46,14 @@ func (m *workerModule) listWorkersHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, workers)
 }
 
+// createWorkerHandler godoc
+// @id createWorker
+// @summary Creates a new wharf-cmd-worker
+// @description Added in v0.8.0.
+// @tags worker
+// @success 200 {object} provisioner.Worker
```
Comment (Alexamakans) [inline pkg/provisionerapi/worker.go:54]: Status is `201 Created` on success.

**Review by Alexamakans — APPROVED** [review]
Good stuff 👍🏻

File: Makefile (hunk 1 above)
Comment (jilleJr) [reply Makefile:23]: Oops. Fixed.

**Review by Alexamakans — APPROVED** [review]
👍🏻

**Pull request merged**